### DISCUSSION TITLE
Nystatefulwidget vs Nypage

### DISCUSSION THREAD

**Discussion by Im-PJ** [nylo-core/nylo#93] [Q&A] [answered]
I have gone through the docs but still didnt understand difference between these two and when one should be used. both looks interchangeable. Im more confused bcoz metro make:page command creates class which extends NyStatefulWidget.

**Comment by agordn52** [comment, marked as answer]
Hi @Im-PJ,

Yeah, you're right.

Both are interchangeable, however, the class that extends `NyStatefulWidget` has more utility/flexibility. 
Hope that helps.

Anthony
